teste2e/author_submit_and_watch_extrinsic_test.go

```diff
@@ -86,6 +86,7 @@ func TestAuthor_SubmitAndWatchExtrinsic(t *testing.T) {
 
 		sub, err = api.RPC.Author.SubmitAndWatchExtrinsic(ext)
 		if err != nil {
+			nonce++
```

Comment:
Why is this required now / how did it work before without this increment?

Reply:
It was all a matter of timing, CI did fail from time to time, I am hoping that this helps.